packages/core/src/CardanoNode/types/CardanoNode.ts

```diff
@@ -30,22 +30,6 @@ export type StakeDistribution = Map<
 >;
 
 export interface CardanoNode {
-  /**
-   * Initialize CardanoNode instance
-   */
-  initialize: () => Promise<void>;
-  /**
-   * Shut down CardanoNode instance
-   *
-   * @throws {CardanoNodeNotInitializedError}
-   */
-  shutdown: () => Promise<void>;
-  /**
-   * Get summaries of all Cardano eras
-   *
-   * @returns {EraSummary[]} Era summaries
-   * @throws {CardanoNodeError}
-   */
```

Comment:
Please describe this breaking change in the commit message

Reply:
Described in 234305e28aefd3d9bd1736315bdf89ca31f7556f @mkazlauskas